app/uk/gov/hmrc/bindingtariffclassification/model/RESTFormatters.scala

```diff
@@ -17,58 +17,61 @@
 package uk.gov.hmrc.bindingtariffclassification.model
 
 import play.api.libs.json._
-import play.json.extra.Jsonx
+import play.json.extra.{InvariantFormat, Jsonx}
 import uk.gov.hmrc.bindingtariffclassification.model.reporting.ReportResult
 import uk.gov.hmrc.play.json.Union
 
 object RESTFormatters {
 
 
   // `Case` formatters
-  implicit val formatCaseStatus = EnumJson.format(CaseStatus)
```

Comment:
I fixed the missing type warnings to match changes made to a similar PR this morning